pulsar-client/src/main/java/com/yahoo/pulsar/client/impl/MessageIdImpl.java

```diff
@@ -77,8 +77,7 @@ public boolean equals(Object obj) {
 
     @Override
     public String toString() {
-        return Objects.toStringHelper(MessageIdImpl.class).add("ledgerId", ledgerId).add("entryId", entryId)
-                .add("partitionIndex", partitionIndex).toString();
+        return String.format("%d:%d:%d", ledgerId, entryId, partitionIndex);
```

Comment:
Sure, but I was just talking about non-partitioned & non-batched.. anyway it's not a big deal and it would be confusing as well.. let's just merge this.